src/Schema/Directives/BaseDirective.php

```diff
@@ -99,7 +125,19 @@ protected function nodeName(): string
      */
     protected function directiveArgValue(string $name, $default = null)
     {
-        return ASTHelper::directiveArgValue($this->directiveNode, $name, $default);
+        if (! isset($this->directiveArgs)) {
+            $this->loadArgValues();
+        }
+
+        return array_key_exists($name, $this->directiveArgs) ? $this->directiveArgs[$name] : $default;
```

Comment:
https://github.com/nuwave/lighthouse/blob/master/CONTRIBUTING.md#multiline-ternary-expressions

```suggestion
        return array_key_exists($name, $this->directiveArgs)
            ? $this->directiveArgs[$name]
            : $default;
```